eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -184,6 +184,25 @@ jobs:
         BuildTargetingString: ${{ parameters.BuildTargetingString }}
         TestMarkArgument: ${{ parameters.TestMarkArgument }}
 
+  - job: 'RunApiStubGen'
+    condition: and(succeededOrFailed(), ne(variables['Skip.ApuStubGen'], 'true'))
+    displayName: 'Run api stub generator'
+    variables:
+    - template: ../variables/globals.yml
+
+    dependsOn:
+      - 'Build'
+
+    pool:
+      vmImage: 'ubuntu-18.04'
+
+    steps:
+    - template: ../steps/run_apistub.yml
+      parameters: 
+        ServiceDirectory: ${{ parameters.ServiceDirectory }}
+        BuildTargetingString: ${{ parameters.BuildTargetingString }}
+        TestMarkArgument: ${{ parameters.TestMarkArgument }}
```

Comment:
api_stub.yml has a `EnvVars`. Do you need to pass that?

Reply:
Removed EnvVars. API parser is not using them now.